File: 20-10-25/app/src/main/java/com/th/btvn/MainActivity.java
package com.th.btvn;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });

        // Initialize buttons
        Button btnExercise1 = findViewById(R.id.btnExercise1);
        Button btnExercise2 = findViewById(R.id.btnExercise2);
        Button btnExercise3 = findViewById(R.id.btnExercise3);

        // Set click listeners
        btnExercise1.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, ShakeDetectionActivity.class);
            startActivity(intent);
        });

        btnExercise2.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, BatteryStatusActivity.class);
            startActivity(intent);
        });

        btnExercise3.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, JsonListViewActivity.class);
            startActivity(intent);
        });
    }
}

File: 08-09-25/app/src/main/java/com/btvn/btvn/MainActivity.kt
package com.btvn.btvn

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }
    }

    fun goToBai1(view : View){
        val intent = Intent(this, bai1Activity::class.java)
        startActivity(intent)
    }

    fun goToBai2(view : View){
        val intent = Intent(this, bai2Activity::class.java)
        startActivity(intent)
    }

    fun goToBai3(view: View){
        val intent = Intent(this, bai3Activity::class.java)
        startActivity(intent)
    }
}